plugin/kprom/config.go

```diff
@@ -101,6 +103,13 @@ func WithClientLabel() Opt {
 	return opt{func(c *cfg) { c.withClientLabel = true }}
 }
 
+// WithStaticLabel adds a static label to all metrics.
```

Comment:
Sorry one last nit -- is it worth noting that the `client_id` const label is always unconditionally set? Should we allow the end user to be able to override it, or should the current behavior be kept?

(separately: good idea on just allowing the user to pass in prometheus.Labels, that's way better than my keyvals suggestion)

Reply:
Yes, the collision of metrics is unpleasant. But I would leave the current behavior, otherwise suddenly someone's  something will break from this change. And according to Occam's razor, new entities should not be invented. And if someone has better ideas, then he can come with an issue or mr